src/optick.h

```diff
@@ -31,20 +31,20 @@
 #include <stddef.h>
 
 #if defined(__clang__) || defined(__GNUC__)
-#	define OPTICK_GCC (1)
-#	if defined(__APPLE_CC__)
-#		define OPTICK_OSX (1)
-#	elif defined(__linux__)
-#		define OPTICK_LINUX (1)
-#	elif defined(__FreeBSD__)
-#		define OPTICK_FREEBSD (1)
-#	endif
+#define OPTICK_GCC (1)

```

Comment:
Blah, I didn't mean to have so many whitespace changes in this file. Darn VS2019 auto-formatter. If this is a big problem I can try to revert it